source/Cosmos.System2/Graphics/VGACanvas.cs

```diff
@@ -30,138 +47,312 @@ public VGACanvas() : base()
             _VGADriver.SetGraphicsMode(ModeToScreenSize(DefaultGraphicMode), (VGADriver.ColorDepth)(int)DefaultGraphicMode.ColorDepth);
         }
 
+        /// <summary>
+        /// Gets or sets the VGA graphics mode
+        /// </summary>
         public override Mode Mode { get; set; }
 
+        /// <summary>
+        /// Clears the screen of all pixels
+        /// </summary>
+        /// <param name="aColor"></param>
         public override void Clear(Color aColor)
         {
             var paletteIndex = _VGADriver.GetClosestColorInPalette(aColor);
             _VGADriver.DrawFilledRectangle(0,0, _VGADriver.PixelWidth, _VGADriver.PixelHeight, paletteIndex);
         }
 
+        /// <summary>
+        /// Disables VGA graphics mode, parent method returns to 80x25 text mode
+        /// </summary>
         public override void Disable()
         {
             if (Enabled)
             {
-                VGAScreen.SetTextMode(TextSize.Size80x25);
```

Comment:
why remove this line?

Reply:
Because FullScreen canvas returns back to 80x25 text mode anyway, no point having the method called twice :)

Reply:
FullscreenCanvas.Disable()

Reply:
cheers :)